Don't look in /usr/local. MacPorts will set the right -I and -L flags.

Do set the right prefix for the library install_name. (The INSTALLLIB
variable was not set anywhere.)
--- Makefile.orig	2011-10-28 02:17:11.000000000 -0500
+++ Makefile	2022-01-14 01:26:51.000000000 -0600
@@ -47,11 +47,7 @@
 LIBSUFFIX=$(shell sh $(BASE)/platform.sh libsuffix)
 VERLIBSUFFIX=$(shell sh $(BASE)/platform.sh libsuffix $(MAJOR))
 DYNLIBFLAG=$(shell sh $(BASE)/platform.sh dynlibflag)
-LIBNAMEFLAG=$(shell sh $(BASE)/platform.sh libnameflag $(MAJOR) $(INSTALLLIB))
-
-# Sane includes
-CFLAGS+=-I/usr/local/include
-LDFLAGS+=-L/usr/local/lib
+LIBNAMEFLAG=$(shell sh $(BASE)/platform.sh libnameflag $(MAJOR) $(PREFIX)/lib)
 
 # Platform so we know what to dlopen
 CFLAGS+=-DPLATFORM_$(PLATFORM)
